app/views/courses/degrees/grade/edit.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= content_for :page_title, "What is the minimum degree classification you require?" %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to degrees_start_provider_recruitment_cycle_course_path %>
+<% end %>
+
+<%= render "shared/errors" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @grade_form, url: degrees_grade_provider_recruitment_cycle_course_path(
+                    @course.provider.provider_code,
+                    @course.provider.recruitment_cycle_year,
+                    @course.course_code,
+                  ),
+                  method: :put,
+                  builder: GOVUKDesignSystemFormBuilder::FormBuilder do |form| %>
+
+      <span class="govuk-caption-xl"><%= course.name_and_code %></span>
+
+      <%= form.govuk_radio_buttons_fieldset :grade, legend: { text: "What is the minimum degree classification you require?", tag: "h1", class: "govuk-heading-xl govuk-!-margin-bottom-2" } do %>
+
+        <%= form.govuk_radio_button :grade, "two_one", label: { text: "2:1 or above (or equivalent)" }, data: { qa: "degree_grade__two_one" }, link_errors: true %>
```

Comment:
I came across this the other day in https://github.com/DFE-Digital/publish-teacher-training/pull/1719/. It’s because in most places, Publish uses manually generated forms and error summaries, which are not compatible with how the form builder does things. I think I ended up using the old error summary partial rather than the one in the form builder to get around this issue.

Reply:
It might just be the time but this is making my head hurt. If you get a sec tomorrow Paul it would be good to run through this please 🙏